Userland/Libraries/LibGfx/Matrix4x4.h

```diff
@@ -37,16 +37,6 @@ constexpr static Vector3<T> transform_point(const Matrix4x4<T>& m, const Vector3
         p.x() * elements[2][0] + p.y() * elements[2][1] + p.z() * elements[2][2] + elements[2][3]);
```

Comment:
@GMTA not sure if you noticed my last message, so here it is again: I think it would be good to add a fixme here if you are not going to remove `transform_point` right away.

Reply:
@sunverwerth Thanks for reminding me; is this what you meant?

Reply:
Yes